### ISSUE TITLE
Ender Dragon Config Not Found?

### ISSUE THREAD

**Issue by RussianSanity** [Ipsis/Woot#339]
So it was brought to my attention by someone who plays a pack I help develop that you cannot capture the Ender dragon with the shard. Said something about how there should be a config for it.

I went searching and found nothing at all under the config/woot folder to enable it. 
I thought maybe it was the pack so I booted up a default, fresh install of only woot. Same thing.
Nothing to be found.
This is the default config I was provided: https://pastebin.com/Xh2g62z7

I also found this guide online of how to enable it in-game: https://imgur.com/a/oDBj2
However when I tried it, it told me this: https://gyazo.com/9b2c845563198f4c1333060214d4cf39

So I'm just wondering if I'm going crazy and maybe missed it or I'm looking in the wrong spot...  Or it just isn't there?
I'm running with version: woot-1.12.2-1.0.1.jar 

Hope to hear back soon!

**Comment by Ipsis** [comment]
The previous versions of Woot for pre-1.12.2 Minecraft handled the Ender Dragon differently and the online guide you saw was the way to enable it.

However in 1.12.2 I completely changed it and the Ender Dragon is enabled by default and you don't need to do anything to configure it anymore.

Can you post your factory_config.json file as well. This contains the extra configuration for mobs in Woot.

I would also check that someone has tried to capture the Ender Dragon and failed, rather than expected it not to work due to how used Woot to be setup.

So in summary, if you haven't disabled it somewhere the EnderDragon should just work without you doing anything .

**Comment by RussianSanity** [comment]
Here's my factory_config.json from the pack: https://pastebin.com/NKTyAj6Q 
And we've tried before but it never worked. Tried more than once, I should add. We just figured it was a sponge issue since... well sponge is invasive to say the least.

**Comment by Ipsis** [comment]
If you have SpongeForge then that is the reason for the boss capture issue.
Although they seem to have fixed it in the last few days.
See the below bug.

https://github.com/SpongePowered/SpongeForge/issues/1912

**Comment by RussianSanity** [comment]
Ah okay, I'll have to gather my partner and update the pack on our end before I can update sponge it seems. They ran a second recommended build I wasn't aware of this last week. Sorry for the trouble! I'll update this (if I can) when we update everything and try it out.
Thank you!

**Comment by demon57730** [comment]
I have the same problem

spongeforge-1.12.2-2611-7.1.0-BETA-3004.jar

I also have a problem with the ghast I capture them but it just gives me some powder no tear

**Comment by demon57730** [comment]
[https://github.com/SpongePowered/SpongeForge/issues/1912#issuecomment-379816379](url)

**Comment by ArmageddonX430** [comment]
I was about to post how I don't have SpongeForge and I also was unable fully program an Ender Dragon into an ender shard, then I read the tooltip where the shard has to stay in your hotbar and I facepalmed.  Loudly.  Anyone else having issues fully programming something, make sure you didn't accidentally do what I did.  ^_^

**Comment by Ipsis** [comment]
I think I only recently updated the tooltip to include the hotbar information, as it was only mentioned in the  guide before. So believe me, you are not the first :)
